examples/nvme/perf/perf.c

```diff
@@ -475,6 +480,21 @@ submit_single_io(struct ns_worker_ctx *ns_ctx)
 	uint64_t		offset_in_ios;
 	int			rc;
 	struct ns_entry		*entry = ns_ctx->entry;
+	uint64_t current_time;
+
+	if (ns_ctx->io_limit) {
+		current_time = spdk_get_ticks();
+		if ((current_time - ns_ctx->io_limit_time) > g_tsc_rate) {
+			ns_ctx->io_limit_count = 0;
+			ns_ctx->io_limit_time = current_time;
+		}
+
+		ns_ctx->io_limit_count++;
```

Comment:
suggest to move this counter update after the next branch check , which means increment the counter only if its actually submitting a command / although no overflow is expected its more readable

Reply:
done